### PR TITLE
Ci/add discord notify

### PR DESCRIPTION
**Pull request by warstrolo**
This PR adds sending a message to the [OKP4 Discord server](https://discord.gg/okp4) (using webhooks) when a new version is released.

### CODE REVIEW

File: .github/workflows/notify-release.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: ${{ secrets.OKP4_DISCORD_WEBHOOK }}
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json"}'
+          data: |-
+            {
+              "avatar_url": "https://avatars.githubusercontent.com/u/98603954?v=4",
+              "username": "Bot Anik",
+              "content": " 🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! 🎉 \n \n 👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.tagName }} \n 👉 Official repo: https://github.com/${{ github.repository }}"
```
Comment (amimart) [inline .github/workflows/notify-release.yml:21]: The tag references are wrong: `github.event.tagName`. Can you please update them ?

**Review by amimart — CHANGES_REQUESTED** [review]

File: .github/workflows/notify-release.yml (hunk 1 above)
Comment (warstrolo) [reply .github/workflows/notify-release.yml]: fixed, forgot to push the commit

File: .github/workflows/notify-release.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: ${{ secrets.OKP4_DISCORD_WEBHOOK }}
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json"}'
+          data: |-
+            {
+              "avatar_url": "https://avatars.githubusercontent.com/u/98603954?v=4",
+              "username": "Bot Anik",
+              "content": " 🚨 A new version of @${{github.repository}} ${{ github.event.release.tag_name }} has been released! 🎉 \n \n 👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.release.tag_name }} \n 👉 Official repo: https://github.com/${{ github.repository }}"
```
Comment (amimart) [inline .github/workflows/notify-release.yml:21]: Please remove unnecessary spaces
```suggestion
              "content": "🚨 A new version of @${{github.repository}} ${{ github.event.release.tag_name }} has been released! 🎉 \n\n 👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.release.tag_name }} \n 👉 Official repo: https://github.com/${{ github.repository }}"
```

**Review by amimart — CHANGES_REQUESTED** [review]

File: .github/workflows/notify-release.yml (hunk 2 above)
Comment (ccamel) [inline .github/workflows/notify-release.yml:21]: Please, remove unnecessary blank spaces as it was done in project [okp4/ui](https://github.com/okp4/ui/blob/main/.github/workflows/notify-release.yml#L21)

```suggestion
              "content": "🚨 A new version of @${{github.repository}} ${{ github.event.release.tag_name }} has been released! 🎉\n\n👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.release.tag_name }}\n👉 Official repo: https://github.com/${{ github.repository }}"
```

**Review by ccamel — CHANGES_REQUESTED** [review]

**Review by hleonardtek — APPROVED** [review]

**Review by ccamel — APPROVED** [review]

**Review by amimart — APPROVED** [review]
👍

**Pull request merged**